src/Common/src/CoreLib/System/MemoryExtensions.Fast.cs

```diff
@@ -17,46 +17,46 @@ namespace System
     public static partial class MemoryExtensions
     {
         /// <summary>
-        /// Returns a value indicating whether the specified <paramref name="value"/> occurs within the <paramref name="span"/>.
+        /// Returns a value indicating whether the specified <paramref name="other"/> occurs within the <paramref name="span"/>.
         /// <param name="span">The source span.</param>
-        /// <param name="value">The value to seek within the source span.</param>
-        /// <param name="comparisonType">One of the enumeration values that determines how the <paramref name="span"/> and <paramref name="value"/> are compared.</param>
+        /// <param name="other">The value to seek within the source span.</param>
+        /// <param name="comparisonType">One of the enumeration values that determines how the <paramref name="span"/> and <paramref name="other"/> are compared.</param>
         /// </summary>
-        public static bool Contains(this ReadOnlySpan<char> span, ReadOnlySpan<char> value, StringComparison comparisonType)
+        public static bool Contains(this ReadOnlySpan<char> span, ReadOnlySpan<char> other, StringComparison comparisonType)
```

Comment:
I think the "other" terminology is used only in Equals/Compare. I think in Contains "value" works better.

@terrajobst, what was the intention here?

Reply:
What about the other APIs that were part of this change: StartsWith/EndsWith/Overlaps?